### ISSUE TITLE
Find VA Forms - Form Detail Landing Pages

### ISSUE THREAD

**Issue by MickinSahni** [department-of-veterans-affairs/va.gov-team#5590]
## Product Outline
[Redesign the Find a VA Form page](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/find-va-forms-redesign#product-outline-find-va-forms---search--detail-landing-pages)

## Description
As a Veteran, I want specific form information so that I can more easily apply for benefits via available online tools or download forms to do so. 

## Hypothesis or Bet
**If** we give Veterans specific information about particular VA forms **then** it'll be easier for them to apply for benefits using available online tools or accurate VA forms. 

## Definition of done
* Form detail landing page(s) are launched

## Tickets to be created

@MickinSahni 👇 

- [x] [Drupal] Create a "node -type" for Form Detail pages (potentially complete? yes, see [comment here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10606#issuecomment-657590054))
- [x] One example of a fleshed detail page for Steve W. (already created, will have 2 examples)
- [x] [Verify integration with updated Forms API #11156](#11156)
- [x] [Front-End] Begin consuming Form-Detail data (i.e. build the static page ~from Lighthouse API~) (#10819)
- [x] Error message for invalid PDF url
- [x] Batch publishing (blocked by #5549 and Transformers) 
  - Batch 1: Publish Find VA Form detail pages #11155
  - Batch 2: Publish Find VA Form detail pages #12213
- [x] Detail page redirects - #5549 
  - [x] submit request 
  - [x] Map urls to be redirected  [#11336](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11336)
  - [x] Implement redirects  
- [x] Remove form detail pages no index #12242 

- related - iterations to find va forms search

**Comment by MickinSahni** [comment]
- Q: does this involve multiple forms?
  - A: Yes. 100s of forms. There are some that have custom pages. (This is related to the convo about whether we should bring folks to the search page or specific form pages.)

- Q: is this going to be a separate launch from the Find VA Forms Search?
  - A: Yes. The search is going to lead users to PDF downloads (as is currently done)

- Constituent tickets (besides #4379)
  - New ticket: define tech approach to integration with Lighthouse - how form data passed through to FE
  - New ticket: implement integration with Lighthouse
  - Tracking VSP IA review (#3836) - no active PW work but we're involved so tracking

**Comment by jenniferlee-dsva** [comment]
referencing this related ticket for Dave's Drupal engineer (on VAMC team) to build the form detail landing page Drupal authoring experience: #5653

**Comment by AkucherGCIO** [comment]
@AkucherGCIO-  The following tickets still need to be created based on Nick/Aruna meeting last week.

1.  [Drupal] Create a "node -type" for Form Detail pages

* Custom Form Landing Pages = Top 30 forms, which contains extra fields versus the less-popular forms
* Can we also add fields "RelatedTo" and "Online Tool" for all forms? Can we be notified when a new form is added into the VA Forms DB?
	
2. [Lighthouse] Being consuming form data from the Lighthouse Forms API
*  Use Drupal's GraphQL API to pull form data from Drupal into the Lighthouse Forms local database
	
3. [Front-End] Begin consuming Form-Detail data from Lighthouse API

**Comment by AkucherGCIO** [comment]
03/04/20 PW Grooming
@jenniferlee-dsva - will be reviewing stakeholders feedback after the Search pages soft launch happens because of the time limitation to be following up with them.

**Comment by MickinSahni** [comment]
need to indicate on form detail page that the downloaded PDF is 404ing. the form page should still exist but need messaging to inform users of why they aren't seeing the PDF link. this would make detail pages consistent with the search (where we're already suppressing 404ing PDF urls)

**Comment by MarciMcGuireGCIO** [comment]
Checking w/Kevin Walsh to see if https://github.com/department-of-veterans-affairs/va.gov-team/issues/9562 was completed in Jun/Jul 2020.

**Comment by MarciMcGuireGCIO** [comment]
@johnhashva - there are 2, possibly 3 outstanding items on this Epic for the Form Detail Landing page,  Since the landing page has been live for a while now, I'm thinking the remaining items should be detached for later development so that the original epic can be closed.  Thoughts?
